.github/workflows/tests_mac.yaml

```
@@ -1,5 +1,10 @@
 name: tests-macos
-on: [push, pull_request]
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: "*"
```

This shouldn't be needed
```suggestion
  pull_request:
```